### ISSUE TITLE
New issue im encountering

### ISSUE THREAD

**Issue by Jxuser9875321** [gui-fkb/twitch-accounts#5]
"Please update to the latest version and try again. If you still encounter issues, feel free to let me know"

Ok I have updated and tried the newest version, Without using a proxy it seems to work (Successfully created 1 account for testing)
I haven't tested proxies as my IPRoyal residential bandwidth is empty now
However I do assume using a IPRoyal residental proxy formatted as http://user:pass@ip:port 
would work correct?

The issue now is trying the follow bot .go file.
I think this is also an issue with the salamoonder API reference (Like the last issue you fixed for account creation), but nothing I tried worked (Im relatively new to API calls still so I honestly cant do much)

Ive attached an image of the console output!
           
Sorry for bugging you about this but Id like to get this working so I can study and learn from it!

Also for the followbot.go file, I was wondering if its possible to have them follow without having to login and use the kasada solver, or is that something thats not possible? Just wondering as it adds up in cost long term, and Im assuming same would be true when I try to use other API librarys.
![2 (2)](https://github.com/user-attachments/assets/2174efb5-c552-423b-81da-4d4b8d00a31c)

**Comment by gui-fkb** [comment]
Hey, thanks for testing the newest version! 

Yeah the pattern http://user:pass@ip:port should work as proxy configuration, but i'm not sure IPRoyal would work because some Proxy services don't work for big companies or streaming services for legal purposes, i'm not sure if IPRoyal is one of them, but i know that oxylabs won't work

I did spent some time trying to adjust the follow bot while updating the account creation flow, but i couldn't make it work again yet, but i will continue looking into it.

As for skipping the Kasada solver for the follow action, unfortunately, that’s not possible. If you inspect Twitch's traffic using developer tools (F12), you’ll notice that the follow request includes integrity check data, which is enforced by their system.

**Comment by Jxuser9875321** [comment]
So for now the follow .go file wont work? Ill try to mess with it myself but I probably wont get anywhere

**Comment by gui-fkb** [comment]
Exactly, for now the follow-bot feature won't work as expected but I will let you know if I manage to fix it.

**Comment by starbez** [comment]
> Exactly, for now the follow-bot feature won't work as expected but I will let you know if I manage to fix it.

Did you manage to fix it

**Comment by gui-fkb** [comment]
Unfortunely, not yet
